docs/source/CppUsage.md

```diff
@@ -88,6 +88,11 @@ convenient accessors for all fields, e.g. `hp()`, `mana()`, etc:
 
 *Note: That we never stored a `mana` value, so it will return the default.*
 
+The following attributes are supported:
+
+-   `native_shared` (on a field): For string fields, this enables the usage of
```

Comment:
Not sure about the name now, since you're using it for both the object api (all start with `native_`) and the base API (the direct function). So I think we need a name that reflects that.. maybe just `shared` ?

Also, please mention here exactly where this currently happens, i.e. in Direct functions and Pack.

Reply:
Renamed to `shared` and made comment verbose.